bindata/services/downloads-service.yaml

```diff
@@ -1,12 +1,10 @@
+# Default 'downloads' service manifest.
+# The 'downloads' route will be pointing to the 'downloads' deployment. 
 apiVersion: v1
 kind: Service
 metadata:
   namespace: openshift-console
   name: downloads
```

Comment:
Should this be renamed to be downoads custom? to make it more clear that this is the customized service downloads, should someone try and use these files as an example of how to use this feature.